### DISCUSSION TITLE
Can you add some tutorial, from zero for making this code into a clearly application?

### DISCUSSION THREAD

**Discussion by dimzy48** [Toinane/colorpicker#111] [Q&A] [answered]
my question is in title, please help me for learning

**Comment by Toinane** [comment, marked as answer]
Hey @dimzy48 If i understand good, you want to learn how to develop an application with Electron?

**Reply by dimzy48** [reply]
i mean like, how to compile your whole code to be an application

**Reply by Toinane** [reply]
You have the steps here: [how to compile](https://github.com/Toinane/colorpicker#how-to-compile)
If you have any errors, ping me here 👍
